packages/worker/src/api/controllers/global/auth.ts

```diff
@@ -70,7 +70,7 @@ async function authInternal(ctx: any, user: any, err = null, info = null) {
 export const authenticate = async (ctx: any, next: any) => {
   return passport.authenticate(
     "local",
-    async (err: any, user: User, info: any) => {
+    async (err: any, user: any, info: any) => {
```

Comment:
How come the removed type reference?

Reply:
I think I changed this to accommodate updates that have since been removed.